esmvaltool/recipes/ref/recipe_model_benchmarking_timeseries_region.yml

```diff
@@ -0,0 +1,342 @@
+    - schlund_manuel
+    - weigel_katja
+  maintainer:
+    - lauer_axel
+  references:
+    - lauer25gmd
+  projects:
+    - dlrmabak
+
+
+datasets:
+  # Dataset to be benchmarked
+  - {dataset: MIROC6, grid: gn, benchmark_dataset: true}
+
+preprocessors:
+  pp_abs:
+    regrid:
+      target_grid: 1x1
+      scheme: linear
+    extract_shape:
+      shapefile: ar6
+      crop: true
+      decomposed: true
+      ids:
+        Name:
+          - N.Europe
+    area_statistics:
+      operator: mean
+    multi_model_statistics:
+      span: overlap
+      statistics:
+        - operator: percentile
+          percent: 10
+        - operator: percentile
+          percent: 90
+      exclude: [reference_dataset, MIROC6]
+
+  pp_abs_200:
+    extract_levels:
+      levels: 20000
+      scheme: linear
+      coordinate: air_pressure
+    regrid:
+      target_grid: 1x1
+      scheme: linear
+    extract_shape:
+      shapefile: ar6
+      crop: true
+      decomposed: true
+      ids:
+        Name:
+          - N.Europe
+    area_statistics:
+      operator: mean
+    multi_model_statistics:
+      span: overlap
+      statistics:
+        - operator: percentile
+          percent: 10
+        - operator: percentile
+          percent: 90
+      exclude: [reference_dataset, MIROC6]
+
+
+  pp:
+    regrid:
+      target_grid: 1x1
+      scheme: linear
+    extract_shape:
+      shapefile: ar6
+      crop: true
+      decomposed: true
+      ids:
+        Name:
+          - N.Europe
+    anomalies:
+      period: month
+      reference:
+        start_year: 2001
+        start_month: 1
+        start_day: 1
+        end_year: 2010
+        end_month: 12
+        end_day: 31
+    area_statistics:
+      operator: mean
+    multi_model_statistics:
+      span: overlap
+      statistics:
+        - operator: percentile
+          percent: 10
+        - operator: percentile
+          percent: 90
+      exclude: [reference_dataset, MIROC6]
+
+
+  pp_200:
+    extract_levels:
+      levels: 20000
+      scheme: linear
+      coordinate: air_pressure
+    regrid:
+      target_grid: 1x1
+      scheme: linear
+    extract_shape:
+      shapefile: ar6
+      crop: true
+      decomposed: true
+      ids:
+        Name:
+          - N.Europe
+    anomalies:
+      period: month
+      reference:
+        start_year: 2001
+        start_month: 1
+        start_day: 1
+        end_year: 2010
+        end_month: 12
+        end_day: 31
+    area_statistics:
+      operator: mean
+    multi_model_statistics:
+      span: overlap
+      statistics:
+        - operator: percentile
+          percent: 10
+        - operator: percentile
+          percent: 90
+      exclude: [reference_dataset, MIROC6]
+
+
+
+diagnostics:
+
+  timeseries:
+    description: Plot "classical" time series of global mean anomalies including a reference dataset.
+    variables:
+      tas: &var_default
+        start_year: 2001
+        end_year: 2014
+        preprocessor: pp
```

Comment:
Can't we have a longer time period covered? I know that with the CMIP6 historical simulations we can only go up to 2014, but we could (and should) start earlier than 2001. HadCRUT5 should cover also earlier years, right?

Reply:
Yes, I can add longer time series (but I would do that last since it increases calculation time). What would you prefer: As long as possible for every variable or a certain, eventually common period [the reanalysis/observations have different start dates, but I guess 1980 should work for most]? Which end date? Is there already one set for CMIP7 simulations?

Reply:
I think the time period that you chose now is fine. We will discuss this a little more next week at the workshop, I think.